pages/trips/[id]/index.js

```diff
@@ -62,20 +62,19 @@ export default function DetailsPage() {
         />
         <h2> {trip.title} </h2>
         {trip.city}, {trip.country}
-        <br></br>
         {trip.startDate} - {trip.endDate}
         <h3>My plans</h3>
         <p>{trip.description}</p>
-        <h3>{`${trip.title} packing list:`}</h3>
+        <h3>{trip.title} packing list:</h3>
         <PackingListForm onHandleAddToPackingList={handleAddToPackingList} />
         {trip.packingList.length === 0 ? (
           <p>
             Your packing list is empty. <br></br> Do you want to add something?
           </p>
         ) : (
           <ul>
-            {trip.packingList.map((item) => {
-              return <li key={item._id}>{item.name}</li>;
+            {trip.packingList.map(({ _id, name }) => {
+              <li key={_id}>{name}</li>;
             })}
```

Comment:
⚠️⚠️⚠️ if you remove the `return`, you have to remove the `{}` as well!

See how your code is not rendering the `<li>`s now :/

Fix:

```js
trip.packingList.map(({ _id, name }) => <li key={_id}>{name}</li>)
```